platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/UsersPropertiesFileValidator.java

```diff
@@ -23,18 +24,23 @@
 
 public class UsersPropertiesFileValidator extends PropertiesFileValidator {
 
+  @VisibleForTesting
   static final String DEFAULT_CERT_USER_USED_MSG =
       "The default certificate user of [%s] was found in [%s].";
 
+  @VisibleForTesting
   static final String DEFAULT_CERT_USER_IS_USING_DEFAULT_PASSWORD_MSG =
       "The default certificate user of [%s] was found in [%s] with default password of [%s].";
 
+  @VisibleForTesting
   static final String DEFAULT_ADMIN_USER_IS_USING_DEFAULT_PASSWORD_MSG =
       "The default admin user of [%s] was found in [%s] with default password of [%s].";
 
+  @VisibleForTesting
   static final String CANNOT_PARSE_PASSWORD_MSG =
       "Unable to determine if [%s] is using insecure defaults. Cannot parse password from [%s].";
 
+  @VisibleForTesting
   static final String USERS_PROPERTIES_FILE_EXISTS_MSG =
       "The users.properties file is present at [%s].";
```

Comment:
❓  Why are all these visible for testing?

Reply:
They were made package-private previously for testing purposes so per @coyotesqrl's suggestion I thought I would annotate them as such.